src/libcore/cell.rs

```diff
@@ -641,6 +641,8 @@ pub struct UnsafeCell<T: ?Sized> {
     ///
     /// This field should not be accessed directly, it is made public for static
     /// initializers.
+    #[deprecated(since = "1.2.0", reason = "use `get` to access the wrapped value or `new` to
```

Comment:
I think this message may be a bit off-kilter due to the newline, could this use `\` at the end of the string to make sure it's parsed as one line?